### ISSUE TITLE
Revert mobile UI redesign for Glitch flavor (or at least make it optional)

### ISSUE THREAD

**Issue by leo60228** [glitch-soc/mastodon#1898]
### Pitch

Commit 206e9593ac2c808fe85177ab156027c226da8cb6 should be reverted, or its changes should be put behind an option similar to the advanced UI.

### Motivation

This commit noticeably reduces the amount of space available for posts to be displayed on mobile. It also subjectively looks worse. Reverting this was proposed upstream with broadly positive feedback from users, but it seems unlikely to be reverted entirely (though it may be tweaked to reduce the biggest issues): https://github.com/mastodon/mastodon/issues/19571

**Comment by ClearlyClaire** [comment]
I agree that overall the previous UI was nicer, but this commit also addressed a lot of issues with the logged-out design, so either way this will require a lot of work. Gargron has finally acknowledged that this design had issues, so we'll hopefully revisit it when our workload gets a little lighter

**Comment by AkazaRenn** [comment]
Not sure if it's possible to make an option to use the old one in user settings, which could be the best solution for now before the mainline comes with a fix.

**Comment by single-right-quote** [comment]
> Not sure if it's possible to make an option to use the old one in user settings, which could be the best solution for now before the mainline comes with a fix.

to my understanding this would entail a nontrivial amount of dev work to accomplish because the old UI (whose mechanics i am admittedly unavailable with) has been entirely removed, and replaced with server-side rendering

**Comment by ClearlyClaire** [comment]
Yeah this sounds like even more work than managing to revert the commit while still keeping something that works well when logged out.

**Comment by Atn-D** [comment]
:gb:  I wonder, because I have trouble understanding.

Currently I use the flavor glitch to put a custom color theme to my instance.

But if I understood correctly, it would be possible to create other flavors than Vanilla and Glitch?
If yes, does it mean that it would be possible to recode the interface of a profile for the visitors (person not connected to the instance)?

Because it sells me a dream, but I did not find any flavor code on github.

——

:fr: Je me demande, car j’ai du mal a comprendre.

Actuelement j’utilise le flavor glitch pour y mettre un thème de couleur custom à mon instance.

Mais si j’ai bien compris, il serait possible de créer d’autres flavor que Vanilla et Glitch ?
Si oui, est-ce que ça veut dire qu’il serait possible de recoder l’interface d’un profil pour les visiteurs (personne non connectée à l’instance) ?

Car ça me vend du rêve, mais je n’ai trouvé aucun code de flavor sur github.

**Comment by ClearlyClaire** [comment]
Yes, you should be able to write a whole new flavor, although flavors don't change how the HTML is rendered, so you can't e.g. make a server-side-rendered version of those pages using flavors.
